lang/cs/Org.Apache.REEF.IMRU.Examples/MapperCount/MapperCount.cs

```diff
@@ -43,7 +49,7 @@ private MapperCount(IIMRUClient imruClient)
         /// Runs the actual mapper count job
         /// </summary>
         /// <returns>The number of MapFunction instances that are part of the job.</returns>
-        public int Run(int numberofMappers)
+        public int Run(int numberofMappers, string outputFile, IConfiguration fileSystemConfig)
```

Comment:
As discussed separately, we should not assume a uniform Filesystem throughout the job. What if evaluator needs to write input to the different filesystem and output to different filesystem. Hence, it is better to pass it along individual tasks.